scene/animation/animation_player.h

```diff
@@ -95,9 +95,9 @@ class AnimationPlayer : public AnimationMixer {
 		}
 		bool operator<(const BlendKey &bk) const {
 			if (from == bk.from) {
-				return to < bk.to;
+				return StringName::AlphCompare()(to, bk.to) < 0;
```

Comment:
```suggestion
				return StringName::AlphCompare()(to, bk.to);
```
Forgot it was a `<` not a ordering return